website/docs/cheatsheet/cheatsheet.md

```diff
@@ -177,6 +177,44 @@ config = ScriptRunConfig(
 run = Experiment(ws, 'PyTorch model training').submit(config)
 ```
 
+## Distributed GPU Training
+
+Adapt your `ScriptRunConfig` to enable distributed GPU training.
+
+```python {3,8-9,12,19}
+from azureml.core import Workspace, Experiment, ScriptRunConfig
+from azureml.core import Environment
+from azureml.core.runconfig import MpiConfiguration
+
+ws = Workspace.from_config()
+compute_target = ws.compute_targets['powerful-gpu']
+environment = Environment.from_conda_specifications('pytorch', 'env.yml')
+environment.docker.enabled = True
+environment.docker.base_image = 'mcr.microsoft.com/azureml/openmpi3.1.2-cuda10.1-cudnn7-ubuntu18.04'
+
+# train on 2 nodes each with 4 GPUs
+mpiconfig = MpiConfiguration(process_count_per_node=4, node_count=2)
+
+config = ScriptRunConfig(
+    source_directory='<path/to/code>',
```

Comment:
can we add something that specifically says that the `source_directory` is the one that contains `train.py`?

Reply:
added it as a comment in the code itself